_dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue

```diff
@@ -139,6 +182,12 @@ export default {
   data() {
     return {
       loading: false,
+      cssProps: {
```

Comment:
Any reason you don't do it in sass instead ?

Reply:
Places in a separate scss file